frontend/src/components/NewImagePopup.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import "../css/NewImagePopup.css";
+
+const NewImagePopup = ({ onClose, onCreate }) => {
+  const [description, setDescription] = useState("");
+  const [canvasSize, setCanvasSize] = useState("16x16");
+
+  const handleCreate = () => {
+    if (description.trim()) {
+      onCreate({ description, canvasSize });
+      onClose();
+    }
+  };
+
+  const updatePreview = (size) => {
+    setCanvasSize(size);
+  };
+
+  return (
+    <div className="popup-overlay">
+      <div className="popup-box">
+        <h3 className="popup-title">New Image</h3>
+        <div className="popup-line"></div>
+        <div className="popup-form">
+          <label className="popup-label">What would you like to draw?</label>
+          <input
+            type="text"
+            placeholder="Describe what your image will be..."
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+            className="input"
+          />
+          <div className="popup-preview-wrapper">
+            <div className="popup-preview">
+              <div alt="Preview" className="" />
```

Comment:
The image tag didn't have to be removed here, I just meant the image file itself. Sorry about that, maybe we leave the dummy image for now

Reply:
Ok thanks, Also I confused how is it possible to remove it